curvefs/src/metaserver/metaserver_service.cpp

```diff
@@ -20,117 +20,77 @@
  * Author: chenwei
  */
 
-#include "curvefs/src/metaserver/metaserver_service.h"
 #include <list>
 #include <string>
+#include <vector>
+#include <algorithm>
+
+#include "curvefs/src/metaserver/metaserver_service.h"
 
 namespace curvefs {
 namespace metaserver {
-void MetaServerServiceImpl::GetDentry(
-    ::google::protobuf::RpcController* controller,
-    const ::curvefs::metaserver::GetDentryRequest* request,
-    ::curvefs::metaserver::GetDentryResponse* response,
-    ::google::protobuf::Closure* done) {
+
+void MetaServerServiceImpl::DEFINE_RPC(CreateDentry) {
```

Comment:
It is not a good idea like this, beacuse it will sacrifice readability

Reply:
> It is not a good idea like this, beacuse it will sacrifice readability

fixed.